tldr/cross_file_calls/__init__.py

```
@@ -0,0 +1,105 @@
+    "TREE_SITTER_LUA_AVAILABLE",
+    "TREE_SITTER_LUAU_AVAILABLE",
+    "TREE_SITTER_ELIXIR_AVAILABLE",
+    # Internal extraction functions (for backward compatibility)
+    "_extract_file_calls",
+    "_extract_ts_file_calls",
+    "_extract_go_file_calls",
+    "_extract_rust_file_calls",
+    "_extract_java_file_calls",
+    "_extract_c_file_calls",
+    "_extract_php_file_calls",
+]
```

Ruff (0.14.11)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)